### PR TITLE
[ROCm] Re-enabling unit-tests that are now passing on ROCm platform

### PR DESCRIPTION
**Pull request by deven-amd**
/cc @cheshire @chsigg @nvining-work

### CODE REVIEW

File: tensorflow/core/kernels/depthwise_conv_ops_test.cc [hunk 1]
```diff
@@ -102,8 +102,8 @@ TEST_F(DepthwiseConvOpTest, DepthwiseConvHalfCpu) {
   Run<Eigen::half>(Device::CPU);
 }
 
-#ifdef GOOGLE_CUDA
-TEST_F(DepthwiseConvOpTest, DepthwiseConvFloatGpu) { Run<float>(Device::GPU); }
+#if GOOGLE_CUDA || TENSORFLOW_USE_ROCM
+// TEST_F(DepthwiseConvOpTest, DepthwiseConvFloatGpu) { Run<float>(Device::GPU); }
```
Comment (kkimdev) [inline tensorflow/core/kernels/depthwise_conv_ops_test.cc:106]: Q: Why is this commented out?

**Review by kkimdev — APPROVED** [review]

**Comment by gbaned** [comment]
@deven-amd  Can you please check @kkimdev's comments and keep us posted ? Thanks!

File: tensorflow/core/kernels/depthwise_conv_ops_test.cc (hunk 1 above)
Comment (deven-amd) [reply tensorflow/core/kernels/depthwise_conv_ops_test.cc:106]: that is a mistake.  let me fix and push out the updated commit

Comment (deven-amd) [reply tensorflow/core/kernels/depthwise_conv_ops_test.cc]: done

**Comment by deven-amd** [comment]
@cheshire @chsigg gentle ping

**Review by kkimdev — APPROVED** [review]

**Pull request merged**